### ISSUE TITLE
Global toolchain repositories, or installing from a remote URL

### ISSUE THREAD

**Issue by xizhao** [sourcegraph/srclib#175]
It would be nice to have a tc registry (in the beginning just associates git repos w/ languages) so you could do `src tc install rust` to get rust support.

**Comment by sqs** [comment]
Yes, FYI see https://github.com/sourcegraph/srclib/pull/174 for something that will help here.

For now you can of course do `src toolchain get example.com/repo` and it will clone from https://example.com/repo.git.
